### ISSUE TITLE
The README file is mostly empty

### ISSUE THREAD

**Issue by barentsen** [KathrynJones1/catwoman#1]
Hi there! 👋  This issue is opened in relation to openjournals/joss-reviews#2382.

### Issue

The `README` file of this package is mostly empty, i.e. it only contains the package name and logo.  This is a shame because the `README` file is an excellent place to advertise the value of the package.  It is displayed prominently on GitHub and is the first thing users visiting the repository will see. Good README files make packages more accessible.

### Solution

[JOSS recommends](https://joss.readthedocs.io/en/latest/review_criteria.html#documentation) including a high-level overview of the documentation in the `README` file, i.e.
* The statement of need (audience, use case, ...)
* Installation instructions
* Example usage

Of course there is no need to duplicate large sections of the [package docs](https://catwoman.readthedocs.io) in the README.  It is sufficient to provide key links to the docs, accompanied by a few high-level sentences or key examples.

### Bonus tip

On or near line 86 of `setup.py` file, I recommend adding:
```python
long_description=open('README.rst').read(),
```
This will allow the README file to be shown on the [pypi page](https://pypi.org/project/catwoman/) of the package, because pypi renders whatever is in `long_description`.

**Comment by KathrynJones1** [comment]
Hello! Thank you for this issue, I have edited the README document as you have recommended, adding a brief description of catwoman, a link to the documentation and an example of how to install. I considered copying the 'quickstart' example from the documentation but decided against it since it would be just a repeat of the Quickstart tab in the docs - however if you think this is required then let me know and I can add it in. 
I have also added the long_description line to the setup.py file so it shows up on the pypi page, thank you for that recommendation!
